_posts/2020-02-14-running-a-great-socratic-seminar.md

```diff
@@ -0,0 +1,70 @@
+---
+layout: blog
+type: blog
+title: "Running a Great Socratic Seminar"
+permalink: /running-a-great-socratic-seminar/
+---
+
+People often ask: why run a meetup? Isn't it a lot of work? What is the incentive if you aren't being paid?
+
+Shaping a local community around shared values and common interests is quite rewarding; the friendships you build and the impact you have on one another span a lifetime--you cannot put a dollar value on the collective experiences, cultivated connections and acquired knowledge. 
+
+That said, wanting to run a meetup and actually running one are two very different things. Fostering an unwelcoming, stale environment that does not encourage high quality discussion and regular attendance should be avoided at all costs. Be prepared to dedicate a non-trival amount of time aggregating, digesting and preparing content in a stimulating way. 
+
+This blog post focuses on a discussion-focused event format that is derivative of the "socratic seminar,[LINK]" a meetup style that we have had a lot of success with in the tri-state (NY/NJ/CT) Bitcoin community. The community driven nature of these events stems from the desire to not only encourage participation from attendees and keep content fresh, but to also break reliance on outside presenters for content.  
```

Comment:
with in the tri-state -> within the tri-state

Reply:
```suggestion
This blog post describes a discussion-focused event format that is derivative of the "socratic seminar,[LINK]" a meetup style that we have had a lot of success within the tri-state (NY/NJ/CT) Bitcoin community. The community driven nature of these events stems from the desire to not only encourage participation from attendees and keep content fresh, but to also break reliance on outside presenters for content.  
```